.gersemirc

```diff
@@ -0,0 +1,6 @@
+color: false
+definitions: []
+line_length: 150
```

Comment:
```suggestion
line_length: 80
```

Or 100?

Reply:
I think 100 is a good balance. Switching from 100 to 80 adds a lot more lines.